src/main/java/com/exadel/discountwebapp/category/service/impl/CategoryServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.exadel.discountwebapp.category.service.impl;
+
+import com.exadel.discountwebapp.category.entity.Category;
+import com.exadel.discountwebapp.category.repository.CategoryRepository;
+import com.exadel.discountwebapp.category.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CategoryServiceImpl implements CategoryService {
+
+    private final CategoryRepository categoryRepository;
+
+    @Autowired
+    public CategoryServiceImpl(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
```

Comment:
This constructor can also be created by lombok

Reply:
done